File: 10. Prime and Composite Numbers - CountFactors/answer.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(n):
    # write your code in Python 3.6i = 1
    result = 0
    i = 1
    while (i * i < n):
        if (n % i == 0):
            result += 2
        i += 1
    if (i * i == n):
        result += 1
    return result

File: 5.Prefix Sums - CountDiv/answer.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A, B, K) {
    // write your code in JavaScript (Node.js 8.9.4)
    let curDivisible = A;
    while(curDivisible <= B){
        if(curDivisible % K == 0)
            break
        else
            curDivisible += 1
    }
    
    return Math.floor((B - curDivisible) / K) + 1
}

// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A, B, K) {
    // write your code in JavaScript (Node.js 8.9.4)
    if(A % K == 0)
        return parseInt( (B-A) / K) +1
    else
        return parseInt((B - (A - A % K)) / K) 
}
